engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/CopyWorkflowOutputsActor.scala

```diff
@@ -48,9 +48,22 @@ class CopyWorkflowOutputsActor(workflowId: WorkflowId, override val ioActor: Act
 
   private def copyWorkflowOutputs(workflowOutputsFilePath: String): Future[Seq[Unit]] = {
     val workflowOutputsPath = buildPath(workflowOutputsFilePath)
-
     val outputFilePaths = getOutputFilePaths(workflowOutputsPath)
 
+    // Check if there are duplicated destination paths and throw an exception if that is the case.
+    val destPaths = outputFilePaths.map {case (_ , dest) => dest}
+    val destPathOccurrences = destPaths.groupBy(identity).mapValues(_.length)
+    val duplicatedDestPaths = destPathOccurrences.filter{case (_, occurrences) => occurrences > 1}.keys.toSeq
+    if (duplicatedDestPaths.nonEmpty) {
+      val collidingCopyOptions = outputFilePaths.filter{
+        case (_ ,destination) => duplicatedDestPaths.contains(destination)
+          // Sort by destination path so collides are easy to spot in the exception message
+      }.sortBy { case (_, destination) => destination.pathAsString}
+      val formattedCollidingCopyOptions = collidingCopyOptions.map {case (source, dest) => s"${source.pathAsString} -> ${dest.pathAsString}"}
+      throw new IllegalStateException(
+        "Cannot copy output files to given final_workflow_outputs_dir" +
+          s" as multiple files will be copied to the same path: \n${formattedCollidingCopyOptions.mkString("\n")}")}
```

Comment:
I'm not sure fail-slow name collision handling is going to be okay with everyone looking for a solution to #1641